homeassistant/components/tuya/switch.py

```diff
@@ -299,6 +299,11 @@
             icon="mdi:account-lock",
             entity_category=EntityCategory.CONFIG,
         ),
+        SwitchEntityDescription(
+            key=DPCode.LIGHT,
+            name="Light",
+            entity_category=EntityCategory.CONFIG,
+        ),
```

Comment:
Lights should not be added as switch entities, but as light entities.

Reply:
This just turns the backlight on device controls (as well as the temperature display) on and off. It's not a true light in any sense of the word. There are a number of precedents in HA for such things to be exposed as switches (especially since they are device configuration) - KASA integration, incoming Unifi Protect integration, some Zigbee devices through MQTT, etc.

That said, I can switch to light platform if that makes more sense for Tuya.

Reply:
We already have precedence for this integration specifically, for example:

```py
    # Air conditioner
    # https://developer.tuya.com/en/docs/iot/categorykt?id=Kaiuz0z71ov2n
    "kt": (
        TuyaLightEntityDescription(
            key=DPCode.LIGHT,
            name="Backlight",
        ),
    ),
```

Please adjust it to be a light.